pkg/controller/infinispan/util/cluster.go

```diff
@@ -153,6 +156,62 @@ func (c Cluster) ExistsCache(cacheName, secretName, podName, namespace, protocol
 	}
 }
 
+// ExistsCacheWithAuth returns true if cacheName cache exists on the podName pod
+func (c Cluster) ExistsCacheWithAuth(user, pass, cacheName, podName, namespace, protocol string) (bool, error) {
```

Comment:
@rigazilla looks like **user** var is not using here. Is it correct or not?

Reply:
@rigazilla may be you've missed this comment due to large number of interactions?

Reply:
@dmvolod yeah, thanks. my fault. should bo ok now

Reply:
yes, looks good